addons/l10n_in/models/account_invoice.py

```diff
@@ -67,6 +68,22 @@ def _compute_l10n_in_state_id(self):
             else:
                 move.l10n_in_state_id = False
 
+    @api.depends('partner_id', 'partner_shipping_id', 'company_id', 'l10n_in_state_id')
+    def _compute_fiscal_position_id(self):
+        in_moves = self.filtered(lambda move: move.country_code == 'IN' and move.journal_id.type == 'sale')
```

Comment:
Why not the case for purchase journals?  Do not we need this logic on sale orders/purchase orders as well?

Reply:
```suggestion
        in_moves = self.filtered(lambda move: move.country_code == 'IN' and move.country_id.code == 'IN' and move.journal_id.type == 'sale')
```
@william-andre

Reply:
don't do with move.journal_id.type
maybe it will affect the POS closing entry.
so use `is_invoice` with `include_receipts=True`